dapp/src/sagas/bridge.js

```diff
@@ -232,6 +232,17 @@ function * watchBridgeTransfers () {
   }
 }
 
+function * watchApproveToken () {
+  const communityAddress = yield select(getCommunityAddress)
+  const { foreignTokenAddress, homeTokenAddress } = yield select(state => getCurrentCommunity(state, communityAddress))
+  const { homeTokenAddress: currentHomeToken } = yield select(state => state.screens.dashboard)
+  yield put(actions.getTokenAllowance(foreignTokenAddress, 'foreign'))
+  yield put(actions.getTokenAllowance(homeTokenAddress, 'home'))
+  if (currentHomeToken) {
```

Comment:
what's `currentHomeToken`?

Reply:
If it's a community that has been with bridge v2 I store the home token under `state.screens.dashboard`